python/tests/rd_tests/test_sum.py

```diff
@@ -756,6 +734,49 @@ def test_resample_extrapolate(self):
                     rd_sum.get_interp_direct(key_rate, t),
                 )
 
+    def test_pandas2_compatibility_dataframe_index(self):
+        # regression test to verify that pandas frames in pandas 2
+        # does not break due to missing collection for time_index
+        path = os.path.join(self.TESTDATA_ROOT, "local/ECLIPSE/cp_simple3/SHORT.UNSMRY")
+        smry = Summary(path)
+        try:
+            smry.pandas_frame(
```

Comment:
There is no need to catch this exception and call pytest.fail in the except clause as an uncaught exception is already correctly handled by the test runner.

Reply:
Removed the test since the same issue is discovered by test_summary_to_pandas_frame